src/main/main.ts

```diff
@@ -900,22 +904,22 @@ function loadInstallerMixpanelUUID(): string {
   }
 }
 
-async function relaunchHeadless() {
-  await stopHeadlessProcess();
+async function relaunchHeadless(reason: string | null) {
+  await stopHeadlessProcess(reason);
   initializeHeadless();
 }
 
-async function quitAllProcesses() {
-  await stopHeadlessProcess();
+async function quitAllProcesses(reason: string | null) {
+  await stopHeadlessProcess(reason);
```

Comment:
```suggestion
async function quitAllProcesses(reason: string = "default") {
  await stopHeadlessProcess(reason);
```
IDK what's mean `null` on this parameter, but IMO we can use default value instead null